lib/identity-provider.js

```diff
@@ -105,6 +139,57 @@ IdentityProvider.prototype.create = function (options, cert, callback) {
   })
 }
 
+function createPreferences (options) {
+  var graph = $rdf.graph()
+  var pref = options.url + 'Preferences/prefs.ttl'
+
+  graph.add(
+    sym(pref),
+    sym('http://www.w3.org/1999/02/22-rdf-syntax-ns#type'),
+    sym('http://www.w3.org/ns/pim/space#ConfigurationFile'))
+
+  graph.add(
+    sym(pref),
+    sym('http://purl.org/dc/terms/title'),
+    lit('Preferences file'))
+
+  graph.add(
+    sym(options.card),
+    sym('http://www.w3.org/ns/pim/space#preferencesFile'),
+    sym(pref))
+
+  graph.add(
+    sym(options.card),
+    sym('http://www.w3.org/1999/02/22-rdf-syntax-ns#type'),
+    sym('http://xmlns.com/foaf/0.1/Person'))
+
+  options.defaultContainers.forEach(function() {
+    // TODO 
+  })
+
+  return graph
+}
+
+function createWorkSpace (options, workspace) {
+  var resource = options.url + workspace.name + '/'
+  var acl = resource + options.suffixAcl
+
+  var aclGraph = createAcl(
+    acl,
+    resource,
+    options.agent,
+    options.email)
```

Comment:
No need to create the acls anymore for the workspaces. Except for Inbox which is append-only.